linux_os/guide/services/ssh/ssh_server/sshd_set_maxstartups/tests/comment.fail.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
The profiles in these test scenarios should be cis. I get errors when trying the test suite.
` # profiles = xccdf_org.ssgproject.content_profile_cis`

By the way, at the moment the tests in the `tests/' directory are not run by our CI. We hope to set this up soon.
So these need to be run locally for now, you can check how to set the Test Suite here:
https://complianceascode.readthedocs.io/en/latest/tests/README.html#

Reply:
I've switched this over to the `cis` profile now. I'm going to get a Podman-based environment set up for testing like you suggest. Thanks for the link!